app/models/miq_group.rb

```diff
@@ -206,7 +243,33 @@ def ordered_widget_sets
     end
   end
 
+  def self.create_tenant_group(tenant)
+    tenant_full_name = (tenant.ancestors.map(&:name) + [tenant.name]).join("/")
```

Comment:
# ==> "Red Hat / Cloud / Manageiq / Cruise Control"

I had `name` before, but it is not unique